lang/en/validation.php

```diff
@@ -90,6 +90,7 @@
         'numeric' => 'The :attribute must not be greater than :max.',
         'string' => 'The :attribute must not be greater than :max characters.',
     ],
+    'max_digits' => 'The :attribute must not have more than :max digits.',
```

Comment:
To match the `max.string` and `digits` rules:
```suggestion
    'max_digits' => 'The :attribute must not be greater than :max digits.',
```

Reply:
"be greater" in a numeric context suggests the value of the number, not its length

Reply:
Yeah already thought so, probably better as is then.